cli.js

```diff
@@ -20,4 +21,9 @@ cli.command('verify [car] [cid]', 'check the car piece cid is correct')
 cli.command('find <piece>', 'resolve url for piece cid')
   .action(find)
 
+cli.command('v2 [pieceCid]', 'convert PieceCIDv1 to PieceCIDv2')
```

Comment:
It is required right?

```suggestion
cli.command('v2 <pieceCid>', 'convert PieceCIDv1 to PieceCIDv2')
```